br/pkg/lightning/restore/meta_manager.go

```diff
@@ -605,6 +621,74 @@ func (m *dbTaskMetaMgr) CheckClusterSource(ctx context.Context) (int64, error) {
 	return source, nil
 }
 
+// checkEmptyRegion checks if there are too many empty regions on each store.
+func (m *dbTaskMetaMgr) checkEmptyRegion(ctx context.Context) error {
+	var result pdapi.RegionsInfo
+	if err := m.pdTLS.GetJSON(ctx, pdEmptyRegions, &result); err != nil {
```

Comment:
I'm not sure whether it would returns a `RegionsInfo` struct.

Reply:
Yes, I tested it and it is indeed a `RegionsInfo` struct.